keyboards/melody96/rules.mk

```diff
@@ -24,3 +24,4 @@ NKRO_ENABLE = yes		# USB Nkey Rollover - if this doesn't work, see here: https:/
 BACKLIGHT_ENABLE = yes  # Enable keyboard backlight functionality
 AUDIO_ENABLE = no
 RGBLIGHT_ENABLE = yes
+LTO_ENABLE = yes # reduce firmware size
```

Comment:
We generally prefer to leave this option up to the user. If you want to enforce it for the `via` keymap it should go in `keyboards/melody96/keymaps/via/rules.mk`.

```suggestion
```